peekingduck/pipeline/nodes/input/recorded.py

```diff
@@ -27,7 +28,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
         super().__init__(config, node_path=__name__)
         self._allowed_extensions = ["jpg", "jpeg", "png", "mp4", "avi", "mov"]
         input_dir = config['input_dir']
-        self._resolution = config['resolution']
+        self.resize_info = config['resize']
         self._mirror_image = config['mirror_image']
 
```

Comment:
Possible to log the old resolution here (always) and log the resized resolution (if resized)? I find this useful in past deployments.

Reply:
Added logging. Pylint returns failure for the logs behind too similar... so I changed the wording a little.